paddle/fluid/framework/details/build_strategy.h

```diff
@@ -73,6 +73,10 @@ struct BuildStrategy {
 
   bool fuse_broadcast_op_{false};
 
+#ifdef PADDLE_WITH_CUDA
```

Comment:
Remove this macro, so we can give the same `API` between GPU and CPU.

Reply:
Done. Thanks.